ports/esp32s2/boards/hiibot_iots2/mpconfigboard.mk

```diff
@@ -0,0 +1,17 @@
+USB_VID = 0x239A
+USB_PID = 0x80E8
```

Comment:
0x239A is Adafruit's USB VID. Please change this to one you are allowed to use. If the board is open source, then you can get one from https://pid.codes.